config/intellij-idea-inspections.xml

```diff
@@ -332,7 +332,7 @@
     </inspection_tool>
     <inspection_tool class="ClassWithOnlyPrivateConstructors" enabled="true" level="WARNING" enabled_by_default="true" />
     <inspection_tool class="ClassWithTooManyDependencies" enabled="true" level="ERROR" enabled_by_default="true">
-        <option name="limit" value="34" />
+        <option name="limit" value="36" />
```

Comment:
@romani We just upped this number because we couldn't find a way to suppress it for a specific file, and here we are upping it again.
Should we just turn it off and report problem to IntelliJ?